README.md

```diff
@@ -88,6 +88,15 @@ Also note, that if you call this function too quickly, Google detects that it is
 </svg>
 </details>
 
+## Using a Fork
```

Comment:
This needs to be removed.

Reply:
Restored in dfdb3593e1e2388020736c362ba2bd21d4b6222e